custom-namespaces.yaml

```diff
@@ -1995,3 +1995,15 @@ zoom:
       type: table_view
       tables:
         - table: mozdata.zoom.users
+device_manufacturer_partnerships:
+  glean_app: false
+  pretty_name: Device Manufacturer Partnerships
+  owners:
+    - mbowerman@mozilla.com
+    - kik@mozilla.com
+  spoke: looker-spoke-default
+  views:
```

Comment:
You can also define an explore and measure here so you don't have to do it again in looker-spoke-default:

- measures example: https://github.com/mozilla/lookml-generator/blob/0f699e0bd604e972336293feea5c27f3a7688aa2/custom-namespaces.yaml#L1313-L1320
- explore example: https://github.com/mozilla/lookml-generator/blob/0f699e0bd604e972336293feea5c27f3a7688aa2/custom-namespaces.yaml#L1456-L1460